source/Stargate-Application-Configuration-Tests/ApplicationConfigurationPluginAPITest.class.st

```diff
@@ -42,6 +42,41 @@ ApplicationConfigurationPluginAPITest >> requiredPermissions [
 	^ #('read:application-configuration')
 ]
 
+{ #category : #tests }
```

Comment:
The first tests look very similar. Could some part of them be extracted to a superclass or a helper object?